guide/command-handling/README.md

```diff
@@ -99,7 +99,38 @@ for (const file of commandFiles) {
 }
 ```
 
-As for setting up your files, that's it for now. In the next chapter, you'll learn how to make your commands execute dynamically!
+## Dynamically executing commands
+
+With your `client.commands` Collection setup, you can use it to retrieve and execute your commands! Inside your `message` event, delete your `if`/`else if` chain of commands and replace it with this:
+
+```js {7-14}
+client.on('message', message => {
+	if (!message.content.startsWith(prefix) || message.author.bot) return;
+
+	const args = message.content.slice(prefix.length).trim().split(/ +/);
+	const command = args.shift().toLowerCase();
+
+	if (!client.commands.has(command)) return;
+
+	try {
+		client.commands.get(command).execute(message, args);
+	} catch (error) {
+		console.error(error);
+		message.reply('there was an error trying to execute that command!');
```

Comment:
```suggestion
		message.reply('There was an error trying to execute that command!');
```

Reply:
Since `.reply()` replies with a "@User, text..." format, it's grammatically correct to have this lowercase

Reply:
To be changed in a future PR for v13 then?

Reply:
Yeah, sure. There are a few other usages of `.reply()` around the guide that would need to be updated too